root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp

```diff
@@ -6,6 +6,6 @@
     my $SubsystemSftp = $sshd{"SubsystemSftp"} || "yes";
 
     $OUT = ($SubsystemSftp eq "yes") ? 
```

Comment:
As @gsanchietti would say: _I prefer IFs over the ternary operator_, but it's a matter of taste...

Reply:
ok done